extensions/sql-database-projects/package.json

```diff
@@ -45,6 +45,14 @@
             "command": "sqlDatabaseProjects.open"
           }
         ]
+    },
+    "views": {
+      "explorer": [
```

Comment:
does the Database Project view only show when a database project is open?  The screenshot looks like the viewlet may be getting a bit crowded so we should be sure to only be showing things that are relevant to the user in the workflow they're completing.

Reply:
I'm not spotting anything in the where clause documentation that allows it to trigger off of workspace contents (similar to extension activation events).  I may be able to do something custom by exposing a config value that checks the current workspace, but I think that can be deferred...

I'm toying with the idea of an experience where generalized project support is built into core, then the extensions would contribute capabilities specific to a given project type (e.g. one extension for .sqlproj, one extension for .mlproj, etc.), so I don't think it's worth investing too much into hiding this particular view right now.  Thoughts?